StyleCop.Analyzers/StyleCop.Analyzers.Test/NamingRules/SA1303UnitTests.cs

```diff
@@ -174,6 +174,17 @@ public class FooInner
         }
 
         [TestMethod]
+        public async Task TestConstFieldStatingWithUnderscore()
```

Comment:
* :exclamation: This should read `Starting`
* :exclamation: This should include a comment in the code that SA1303 should not be reported because this issue is reported by SA1309